_topic_maps/_topic_map_rosa.yml

```diff
@@ -116,13 +124,97 @@ Topics:
   File: rosa-deleting-cluster
 - Name: Command quick reference for creating clusters and users
   File: rosa-quickstart
+- Name: Setting up accounts and clusters using AWS Security Token Service (STS)
+  Dir: rosa_getting_started_sts
+  Distros: openshift-rosa
+  Topics:
+  - Name: Understanding the ROSA with STS deployment workflow
+    File: rosa-sts-getting-started-workflow
+  - Name: AWS prerequisites for ROSA with STS
+    File: rosa-sts-aws-prereqs
+  - Name: Required AWS service quotas
+    File: rosa-sts-required-aws-service-quotas
+  - Name: Setting up the environment with STS
+    File: rosa-sts-setting-up-environment
+  - Name: Creating a ROSA cluster with STS quickly
+    File: rosa-sts-creating-a-cluster-quickly
+  - Name: Creating a ROSA cluster with STS using customizations
+    File: rosa-sts-creating-a-cluster-with-customizations
+  - Name: About IAM resources for ROSA with STS
+    File: rosa-sts-about-iam-resources
+  - Name: Interactive cluster creation mode reference
+    File: rosa-sts-interactive-mode-reference
+  - Name: Accessing a ROSA cluster
+    File: rosa-sts-accessing-cluster
+  - Name: Configuring identity providers using the OCM console
+    File: rosa-sts-config-identity-providers
+  - Name: Revoking access to a ROSA cluster
+    File: rosa-sts-deleting-access-cluster
+  - Name: Deleting a ROSA cluster
+    File: rosa-sts-deleting-cluster
 ---
-Name: Red Hat OpenShift Cluster Manager
-Dir: ocm
+Name: Cluster administration
```

Comment:
I'm not moving any of the assemblies or sub-directories yet for the Cluster administration book. 

We are running into a road block with moving directories around to accommodate individual topic maps in a shared repo.